soc/rockchip/rk3588s/Kconfig.defconfig.rk3588s

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 Université Gustave Eiffel
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_RK3588S
+
+config NUM_IRQS
+	int
```

Comment:
this needs to be fixed in the first commit, not added wrong then fixed in a later commit